### DISCUSSION TITLE
so-import-pcap on standalone install: no suricata analysis of pcap

### DISCUSSION THREAD

**Discussion by jpl33** [Security-Onion-Solutions/securityonion#2489] [Unsupported Versions] [answered]
I have a standalone install on virtualBox.
when I run so-import-pcap I only get the zeek logs, but no suricata output. 
when I run the same file through 16.04.7 securityOnion, I get plenty of snort (ET) events.
I tried digging into it and found:
1.  /etc/suricata is missing although so-import-pcap and the basic command-line for suricata refer to it. I tried copying /opt/so/conf/suricata to /etc and restart suricata - no joy.
2. I am attaching the content of /opt/so/log/suricata/import/<import_code>/console.log:
27/12/2020 -- 21:03:08 - <Notice> - This is Suricata version 5.0.4 RELEASE running in USER mode
27/12/2020 -- 21:03:08 - <Error> - [ERRCODE: SC_ERR_FOPEN(44)] - Error opening file: "./stats.log": Permission denied
27/12/2020 -- 21:03:08 - <Warning> - [ERRCODE: SC_ERR_INVALID_ARGUMENT(13)] - output module "stats": setup failed
27/12/2020 -- 21:03:08 - <Warning> - [ERRCODE: SC_WARN_NO_STATS_LOGGERS(261)] - stats are enabled but no loggers are active
27/12/2020 -- 21:03:15 - <Error> - [ERRCODE: SC_ERR_FOPEN(44)] - /input.pcap: Permission denied
27/12/2020 -- 21:03:15 - <Warning> - [ERRCODE: SC_ERR_PCAP_DISPATCH(20)] - Failed to init pcap file /input.pcap, skipping
27/12/2020 -- 21:03:15 - <Notice> - all 1 packet processing threads, 2 management threads initialized, engine started.
27/12/2020 -- 21:03:15 - <Error> - [ERRCODE: SC_ERR_INVALID_ARGUMENT(13)] - pcap file reader thread failed to initialize
27/12/2020 -- 21:03:15 - <Notice> - Signal Received.  Stopping engine.

seems to be a permissions issue somewhere...

**Comment by dougburks** [comment]
I just tested using the pcaps from https://www.malware-traffic-analysis.net/2020/12/28/index.html and it appears to be working correctly for me: 
![image](https://user-images.githubusercontent.com/1659467/103313441-11867280-49ee-11eb-9a3d-8ca4c0fd3a45.png)

What permissions do you have on the pcap file that you're trying to import?

**Comment by jpl33** [comment]
the permissions used to be root:vboxsf because I ran them from a shared folder. 
I chown-ed the files to security:security, copied them to my home directory, and imported them - no good.
tried the same with the malware-traffic-analysis PCAPs - no good.

doug - you haven't said anything about the missing /etc/suricata folder - doesn't that count for something?
Oh please don't tell me I have to re-install. again!  :-(

**Comment by dougburks** [comment, marked as answer]
We don't use /etc/suricata, so you don't need to worry about that.

You'll want to ensure that the `suricata` user can read the pcap. You might want to try moving the pcap to /tmp/ and then chmod-ing the file to make it world-readable.

Alternatively, if your VM has Internet access, you could try downloading a pcap directly into the /tmp/ directory like this:
```
cd /tmp
wget https://www.malware-traffic-analysis.net/2020/09/02/2020-09-02-Emotet-epoch-1-with-Trickbot-gtag-mor119.pcap.zip
unzip 2020-09-02-Emotet-epoch-1-with-Trickbot-gtag-mor119.pcap.zip
sudo so-import-pcap 2020-09-02-Emotet-epoch-1-with-Trickbot-gtag-mor119.pcap
```

**Comment by jpl33** [comment]
BINGO!!
works. I finally got suricata alerts when I imported the pcap from /tmp.
so now what?
I'll add "suricata" to my "security" group
I just wanted to import pcap, so I could "kick the tires" of SO2.
Now I can get on with it.
Thank You!!!!
